HW_1/task1.py

```diff
@@ -0,0 +1,13 @@
+# Найдите сумму цифр трехзначного числа.
+n = int(input('Введите трехзначное число: '))
+while n // 1000 > 0 :
+    if n//1000 > 0:
+        print('Число не трехзначное')
+        n = int(input('Введите трехзначное число: '))
```

Comment:
Юрий, зачем это условие? Оно ведь уже у вас есть в определении цикла `while`.
```suggestion
    print('Число не трехзначное')
    n = int(input('Введите трехзначное число: '))
```

Reply:
Четырехзначные числа и более цикл пропустит.